src/js/_enqueues/admin/password-toggle.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Adds show/hide toggle for the setup config database password.
+ *
+ * @since 6.2.0
+ * @output wp-admin/js/setup-config.js
```

Comment:
```suggestion
 * @output wp-admin/js/password-toggle.js
```

Reply:
This has been updated with [ab58f95](https://github.com/WordPress/wordpress-develop/pull/3725/commits/ab58f95532c22d4d24af6dad9b9a01e8ace005c2). (I failed to see that you could "commit suggestion" here in GitHub)